backend/YouTubeV2.Application/EntityConfiguration/UserEntityConfiguration.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using YouTubeV2.Application.Model;
+
+namespace YouTubeV2.Application.EntityConfiguration
+{
+    public class UserEntityConfiguration : IEntityTypeConfiguration<User>
+    {
+        public void Configure(EntityTypeBuilder<User> builder)
+        {
+            builder.HasKey(x => x.Id);
+
+            builder.HasIndex(x => x.NormalizedEmail).IsUnique();
+            builder.HasIndex(x => x.NormalizedUserName).IsUnique();
+
+            builder.Property(x => x.Name);
```

Comment:
I guess based on constants file name has some max length maybe use it here. Same with surname, username etc